docker/uuvsim/Dockerfile

```diff
@@ -102,15 +114,15 @@ RUN mkdir -p uuv_ws/src/uuv_simulator
 
 # Copy the VRX repository from the local file system
 # We can't use the USER:GROUP variables until Docker adds support to --chown
-COPY --chown=developer:developer . uuv_ws/src/uuv_simulator
+#COPY --chown=developer:developer . uuv_ws/src/uuv_simulator
 
 # Compile the VRX project.
-RUN /bin/bash -c ". /opt/ros/${DIST}/setup.bash && cd uuv_ws && catkin_make"
+#RUN /bin/bash -c ". /opt/ros/${DIST}/setup.bash && cd uuv_ws && catkin_make"
```

Comment:
This avoids the need to pull in files from the host system in order to build the project in the image. A drawback is that the image won't contain a pre-built version of the system; the user will have to run catkin_make first before it can be used. I think this is good for our use-case but if we ever want to put the image in a repository so non-developers can just run it we will probably need to change this.